mysite/_config.php

```diff
@@ -6,10 +6,11 @@
 global $database;
 
 // find the database name from the environment file
-if(defined('SS_DATABASE_NAME') && SS_DATABASE_NAME) {
+if (defined('SS_DATABASE_NAME') && SS_DATABASE_NAME) {
 	$database = SS_DATABASE_NAME;
-} else {
-	$database = 'SS_cwp';
+} elseif (!defined('SS_DATABASE_CHOOSE_NAME') || !SS_DATABASE_CHOOSE_NAME)
+) {
```

Comment:
Can we bring this line up onto the end of the previous line?

Also I feel like this should target the `1` branch so it'll go out in a minor release rather than a patch release - thoughts?

Reply:
True it technically adds functionality, but it's also a bugfix in my opinion.

Reply:
Yeah, but it's more about the way it changes the functionality - i.e. it could be a bug but if it can't be fixed in a way that doesn't cause unexpected changes in behaviour then we can't call it a semver patch change